adjust font-family to the website

s-luong07 opened the pull request:
![font-family](https://user-images.githubusercontent.com/63530290/109109629-7dcddb80-76ea-11eb-983d-543af13c910f.png)
## Adjust font-family to the website 
- adjust the font-family so our website will have similar theme to others Lambda's platform

s-luong07 committed:
adjust font-family to the website

jamesjlundin reviewed (approved):
styling changes look good

jakegrella committed:
Update headerStyle.js

removed comments, added backup 'serif' to font-family: 'Tiempos Headline', serif;

jakegrella committed:
Update landingStyle.js

removed comments

jakegrella committed:
Update room.js

removed comments in styling

jakegrella committed:
Update index.css

removed comments

jakegrella reviewed (approved):
looks good, I removed comments
